### ISSUE TITLE
change tenant connection to system 

### ISSUE THREAD

**Issue by rawdasherif119** [hyn/multi-tenant#924]
i need to get data from one tenant  so i set connection using 
app(Environment::class)->tenant($website);

now i need to set this data to system database but using 
DB::connection('system') not good for me because i need to use eloquent relation to insert this data 
i need to set system connection globally again in the same method 
anyway?

**Comment by bkintanar** [comment]
You can use `Model::on('system')`

**Comment by Dev12230** [comment]
still insert into tenant database  ??
hint : using in model UseTenantConnection

**Comment by Dev12230** [comment]
> You can use `Model::on('system')`
still insert into tenant database ??
hint : using in model UseTenantConnection

**Comment by Dev12230** [comment]
> You can use `Model::on('system')`

**Comment by Dev12230** [comment]
still insert into tenant database 
hint : using in model UseTenantConnection

**Comment by rawdasherif119** [comment]
> You can use `Model::on('system')`
-------------------------------------------------------------
still insert into tenant database 
because i am using UseTenantConnection in model ??

**Comment by fletch3555** [comment]
I don't quite understand why you're doing that @Dev12230.  A model class should use the UsesSystemConnection or UsesTenantConnection trait.  The system connection is ALWAYS available, and the tenant connection is only available if a tenant has been identified.

This means that the two connections exist in parallel and you should have no problem using the system connection at any point assuming your models are configured correctly.

**Comment by bkintanar** [comment]
Closing this issue. If you are still having issues with the package, please create a new issue. Thank you
